samples/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0"?>
```

Comment:
Maybe api-pom.xml would be better? We don't want to convey that this is a "live" pom.xml.